documentation/configuration/parameters/oracleSqlPlus.md

```diff
@@ -51,4 +51,11 @@ flyway {
 <configuration>
     <oracleSqlplus>true</oracleSqlplus>
 </configuration>
-```
\ No newline at end of file
+```
+
+## Use Cases
+
+### Configuring consistent sessions for your migrations
+
+See [site and user profiles](/documentation/database/oracle#site-profiles-gloginsql--user-profiles-loginsql) to learn about how this is achieved with
```

Comment:
May just want to link to the entire sqlplus docs here. This use case implies that sqlplus turns on a specific feature like this one, instead of the whole sqlplus support.

Reply:
see latest commits